client/src/app/app.component.ts

```diff
@@ -1,19 +1,24 @@
-import { Component } from '@angular/core';
+import { Component, HostBinding } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { Router } from '@angular/router';
 import { DecodedToken, Token } from 'src/app/storage/Token';
+import { MatDialog } from '@angular/material/dialog';
+import { Theme, ThemeService } from './services/theme.service.ts.service';
```

Comment:
Import from `src`

Reply:
Fixed in: https://github.com/CMEPW/Smersh/pull/49/commits/69d0cef7dab090e012dcf0784c237b7c1797cb18